.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,66 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [master]
+  schedule:
+    - cron: '0 17 * * 3'
```

Comment:
Do you think we should run this on schedule?

Reply:
I don't think we need to run it on a Schedule

Reply:
Running it on PRs and Master would suffice